File: src/Nethereum.Generators.JavaScript/codeGenNodeTests/MyContractName/DTOs/ApprovalEventDTO.cs
using System;
using System.Threading.Tasks;
using System.Numerics;
using Nethereum.Hex.HexTypes;
using Nethereum.ABI.FunctionEncoding.Attributes;
namespace StandardToken.MyContractName.DTOs
{
    [Event("Approval")]
    public class ApprovalEventDTO
    {
        [Parameter("address", "_owner", 1, true )]
        public string Owner {get; set;}
        [Parameter("address", "_spender", 2, true )]
        public string Spender {get; set;}
        [Parameter("uint256", "_value", 3, false )]
        public BigInteger Value {get; set;}
    }
}

File: src/Nethereum.Generators.JavaScript/codeGenNodeTestsVb/MyContractName/DTOs/ApprovalEventDTO.vb
Imports System
Imports System.Threading.Tasks
Imports System.Collections.Generic
Imports System.Numerics
Imports Nethereum.Hex.HexTypes
Imports Nethereum.ABI.FunctionEncoding.Attributes
Namespace StandardToken.MyContractName.DTOs

    <[Event]("Approval")>
    Public Class ApprovalEventDTO
    
        <[Parameter]("address", "_owner", 1, true)>
        Public Property Owner As String
        <[Parameter]("address", "_spender", 2, true)>
        Public Property Spender As String
        <[Parameter]("uint256", "_value", 3, false)>
        Public Property Value As BigInteger
    
    End Class

End Namespace
